package.json

```diff
@@ -16,6 +16,7 @@
     "refresh-workspace-hosts": "yarn workspace @prairielearn/prairielearn start --refresh-workspace-hosts-and-exit"
   },
   "dependencies": {
+    "eslint-plugin-react": "^7.37.1",
```

Comment:
Let's move this to `devDependencies`.

Reply:
Moved to `devDependencies` in [42543c9](https://github.com/PrairieLearn/PrairieLearn/pull/10800/commits/42543c9a1ca769d8b9f72a66c1f6d1916060f477).